src/main/java/uk/co/drnaylor/mcmmopartyadmin/commands/subcommands/ListPartiesSubCommand.java

```diff
@@ -1,121 +1,125 @@
 package uk.co.drnaylor.mcmmopartyadmin.commands.subcommands;
 
-import com.gmail.nossr50.api.PartyAPI;
-import com.gmail.nossr50.datatypes.party.Party;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+
 import org.bukkit.ChatColor;
 import org.bukkit.OfflinePlayer;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
+
+import com.gmail.nossr50.api.PartyAPI;
+import com.gmail.nossr50.datatypes.party.Party;
+
 import uk.co.drnaylor.mcmmopartyadmin.PartyAdmin;
 import uk.co.drnaylor.mcmmopartyadmin.dualcommandexecutor.DualSubCommandInterface;
 import uk.co.drnaylor.mcmmopartyadmin.locales.L10n;
 
 public class ListPartiesSubCommand implements DualSubCommandInterface {
 
-    private List<String> permissions = new ArrayList<String>();
```

Comment:
Again, your formatting has completely removed any chance I can see of seeing the change. Please make sure the diff is minimal, keep the formatting please.